### PR TITLE
[Issue-49]: Backend likes

### PR DESCRIPTION
**Pull request by ltuffery**
# Description
Like user profile

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Unit tests added (if applicable).
- [x] The code formatted (e.g. with `npm run format`)

### CODE REVIEW

File: api/database/migrations/005_create_likes_table.sql
Comment (swotex) [inline api/database/migrations/005_create_likes_table.sql:1]: Ajouter une date pour l'historique et tri/recherche approfondi ?

File: api/src/Model/Like.php
Comment (swotex) [inline api/src/Model/Like.php:1]: Ajout d'une date ?

File: api/src/Model/User.php
Comment (swotex) [inline api/src/Model/User.php:1]: Ajout de date ?

Comment (ltuffery) [reply api/src/Model/User.php:1]: Ajout de quelle date sur l'utilisation ?

**Review by swotex — APPROVED** [review]
LGFM

**Pull request merged**